resources/views/base/index.blade.php

```diff
@@ -59,7 +59,7 @@
                         @endif
                         <td><a href="/server/{{ $server->uuidShort }}">{{ $server->name }}</a></td>
                         <td>{{ $server->nodeName }}</td>
-                        <td><code>@if(!is_null($server->ip_alias)){{ $server->ip_alias }}@else{{ $server->ip }}@endif:{{ $server->port }}</code></td>
+                        <td style="max-width:290px;word-wrap:break-word;"><code>@if(!is_null($server->ip_alias)){{ $server->ip_alias }}@else{{ $server->ip }}@endif:{{ $server->port }}</code></td>
```

Comment:
Not very elegant in my opinion. style="" attributes are to be avoided in general and this will also only work on large screens. This should be done with media queries and the same different widths bootstrap is using so it will work across various screen sizes.